src/main/webapp/WEB-INF/tags/navBarBrand.tag

```diff
@@ -0,0 +1,5 @@
+<%@ tag description="TEAMMATES logo brand"%>
```

Comment:
`navBarBrand` is a class name taken from Bootstrap, right? It doesn't really tell the purpose of the tag file. `teammatesLogo.tag` is a better name.